.github/workflows/staging-build-macos.yml

```diff
@@ -0,0 +1,52 @@
+name: Build macOS using tar artifacts
+
+on:
+#  schedule:
+#    - cron: '0 12 * * *'
+  repository_dispatch:
+    types: [staging-build-macos]
+
+jobs:
+  build-es-artifacts-x64:
+    name: Build macOS ES Artifacts X64
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Set Up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 14
+
+    - name: Required Packages
+      run: ./release-tools/scripts/required_packages.sh
+
+    - name: Build macOS
+      env:
+        install4j_license: ${{ secrets.INSTALL4J_LICENSE }}
+      run: ./elasticsearch/macos/opendistro-macos-build.sh macos x64
```

Comment:
The script name itself is macos build.sh I dont think we need one more argument stating the same

Reply:
Same discussion as last time, I do not think it is a problem having it.
But I do not against either way here.

Reply:
I included it to be consistent with the existing scripts. Removed it now.